docs/gateway-api/gateway-api-v1.md

```diff
@@ -146,14 +144,12 @@ A sample HTTPRoute object is shown below:
         weight: 2
   ```
 
-The above HTTPRoute object gets translated to two child VS in the AVI controller. One child VS with match criteria as the path begins with `/bar` and a single Pool Group with a single pool and another child VS with match criteria as path begins with `/foo`, a single Pool Group with two pools, and an HTTP Request policy to add `my-header` to the HTTP request forwarded to the backends.
+The above HTTPRoute object gets translated to two child VSes in the AVI controller. One child VS with match criteria as the path begins with `/bar` and a single Pool Group with a single pool and another child VS with match criteria as path begins with `/foo`, a single Pool Group with two pools, and an HTTP Request policy to add `my-header` to the HTTP request forwarded to the backends.
 
-Hostnames are mandatory and cannot contain wildcard.
+Hostnames are mandatory and can be prefixed with a wildcard.
```

Comment:
Are we specifying somewhere `*`  as hostname and empty hostname not accepted in HTTPRoute hostname?

Reply:
Specified in HTTPRoute limitation section